packages/react-admin/src/mui/list/Filter.js

```diff
@@ -69,7 +71,6 @@ export class Filter extends Component {
         const {
             classes = {},
             context,
-            debounce,
```

Comment:
why would you want it to be in the `rest` parameters and pass it to the `<FilterForm>` if it doesn't use it?

Reply:
not sure what you're asking, but debounce is the number of ms to debounce the filter function. This is only used by form and not by button.